shared/resources/styles/components/toasts.scss

```diff
@@ -1,6 +1,6 @@
 .toast-notification {
   position: fixed;
-  z-index: $zindex-modal;
+  z-index: 1051;
   min-height: 50px;
```

Comment:
can you do a ` z-index: $zindex-modal + 1`;  here?  I'd prefer to continue using variables so they'll update if we ever update bootstrap.  The defaults are listed in `styles/variables/tutor-bootstrap.scss`

Reply:
oops, I was testing it and forgot to change it back. will do 👍